core/src/cats/create_bareos_database.in

```diff
@@ -25,7 +25,7 @@
 set -e
 set -u
 
-# change to root dir
+# change to root dir needed to fix pg binary issue
```

Comment:
```suggestion
# avoid misleading psql warning `could not change directory to ...```

Reply:
```suggestion
# avoid misleading postgres warning 'could not change directory to ...'
```
so maybe like this ?

Reply:
done with a "# avoid misleading PostgreSQL warning 'could not change directory to ...'"